changelog/Microsoft.IdentityGovernance.AccessReviews.json

```
@@ -1,5 +1,63 @@
 {
     "changelog": [
+        {
+          "ChangeList": [
+            {
+              "Id": "c749c50d-8b7b-4274-ab46-45a49b9609c3",
+              "ApiChange": "Resource",
+              "ChangedApiName": "accessReviewInstanceDecisionItemAccessPackageAssignmentPolicyResource",
+              "ChangeType": "Addition",
+              "Description": "Added the [accessReviewInstanceDecisionItemAccessPackageAssignmentPolicyResource](https://docs.microsoft.com/en-us/graph/api/resources/accessReviewInstanceDecisionItemAccessPackageAssignmentPolicyResource?view=graph-rest-beta) resource type.",
+              "Target": "accessReviewInstanceDecisionItemAccessPackageAssignmentPolicyResource"
+            },
+            {
+              "Id": "c749c50d-8b7b-4274-ab46-45a49b9609c3",
+              "ApiChange": "Resource",
+              "ChangedApiName": "accessReviewInstanceDecisionItemAzureRoleResource",
+              "ChangeType": "Addition",
+              "Description": "Added the [accessReviewInstanceDecisionItemAzureRoleResource](https://docs.microsoft.com/en-us/graph/api/resources/accessReviewInstanceDecisionItemAzureRoleResource?view=graph-rest-beta) resource type.",
+              "Target": "accessReviewInstanceDecisionItemAzureRoleResource"
+            },
+            {
+              "Id": "c749c50d-8b7b-4274-ab46-45a49b9609c3",
+              "ApiChange": "Resource",
+              "ChangedApiName": "accessReviewInstanceDecisionItemServicePrincipalResource",
+              "ChangeType": "Addition",
+              "Description": "Added the [accessReviewInstanceDecisionItemServicePrincipalResource](https://docs.microsoft.com/en-us/graph/api/resources/accessReviewInstanceDecisionItemServicePrincipalResource?view=graph-rest-beta) resource type.",
+              "Target": "accessReviewInstanceDecisionItemServicePrincipalResource"
+            },
+            {
+              "Id": "c749c50d-8b7b-4274-ab46-45a49b9609c3",
+              "ApiChange": "Resource",
+              "ChangedApiName": "instance",
+              "ChangeType": "Addition",
+              "Description": "Added the **instance** relationship to [accessReviewInstanceDecisionItem](https://docs.microsoft.com/en-us/graph/api/resources/accessReviewInstanceDecisionItem?view=graph-rest-beta) resource.",
+              "Target": "accessReviewInstanceDecisionItem"
+            },
+            {
+              "Id": "c749c50d-8b7b-4274-ab46-45a49b9609c3",
+              "ApiChange": "Resource",
+              "ChangedApiName": "decisions",
+              "ChangeType": "Addition",
+              "Description": "Added the **decisions** relationship to [accessReviewSet](https://docs.microsoft.com/en-us/graph/api/resources/accessReviewSet?view=graph-rest-beta) resource.",
+              "Target": "accessReviewSet"
+            },
+            {
```

These aren't part of your changes
```suggestion
            {
```